tools/osbuild-depsolve-dnf

```diff
@@ -65,14 +65,16 @@ class Solver():
 
         if "name" in desc:
             repo.name = desc["name"]
+
+        # at least one is required
         if "baseurl" in desc:
             repo.baseurl = desc["baseurl"]
         elif "metalink" in desc:
             repo.metalink = desc["metalink"]
         elif "mirrorlist" in desc:
             repo.mirrorlist = desc["mirrorlist"]
         else:
-            assert False
+            assert ValueError("missing either `baseurl`, `metalink`, or `mirrorlist` in repo")
```

Comment:
This should be `raise ValueError()`, no?

Reply:
Yes!